ResearchUXFactory/ORKStep+ResultsExtension.swift

```diff
@@ -198,3 +198,51 @@ extension ORKNumericAnswerFormat : SBAQuestionResultMapping {
         return result
     }
 }
+
+extension ORKTextAnswerFormat : SBAQuestionResultMapping {
+    
+    func questionResult(identifier: String, answer: Any?) -> ORKQuestionResult? {
+        
+        let result = ORKTextQuestionResult(identifier: identifier)
+        if let text = answer as? String {
+            result.textAnswer = text
+        }
+        
+        return result
+    }
+}
+
+extension ORKEmailAnswerFormat : SBAQuestionResultMapping {
+    
+    func questionResult(identifier: String, answer: Any?) -> ORKQuestionResult? {
+        
+        let result = ORKTextQuestionResult(identifier: identifier)
+        if let text = answer as? String {
+            result.textAnswer = text
+        }
+        
+        return result
+    }
+}
+
+extension ORKDateAnswerFormat : SBAQuestionResultMapping {
+    
+    func questionResult(identifier: String, answer: Any?) -> ORKQuestionResult? {
+        
+        let result = ORKDateQuestionResult(identifier: identifier)
+        if let date = answer as? Date {
+            result.dateAnswer = date
+        }
+        
+        return result
+    }
+}
+
+extension ORKHealthKitCharacteristicTypeAnswerFormat: SBAQuestionResultMapping {
+    func questionResult(identifier: String, answer: Any?) -> ORKQuestionResult? {
+        
+        let answerFormat = self.implied() as! SBAQuestionResultMapping
```

Comment:
I think elsewhere this uses a guard and an assert to indicate that there may not be a mapping for the answer format. Could you please do that, instead? thanks.